scipy/spatial/_plotutils.py

```diff
@@ -157,7 +160,8 @@ def voronoi_plot_2d(vor, ax=None, **kw):
         raise ValueError("Voronoi diagram is not 2-D")
 
     if kw.get('show_points', True):
-        ax.plot(vor.points[:,0], vor.points[:,1], '.')
+        point_size = kw.get('point_size', 2.0)
```

Comment:
Default should be `None`, for consistency with user-specific matplotlib defaults.

Reply:
Updated